noir-projects/aztec-nr/easy-private-state/src/easy_private_uint.nr

```diff
@@ -21,23 +21,25 @@ impl<Context> EasyPrivateUint<Context> {
 
 impl<Context> EasyPrivateUint<&mut PrivateContext> {
     // Very similar to `value_note::utils::increment`.
-    pub fn add(self, addend: u64, owner: AztecAddress) {
+    pub fn add(self, addend: u64, owner: AztecAddress, outgoing_viewer: AztecAddress) {
```

Comment:
RIP Klix.
![The Big Lebowski Goodnight Sweet Prince GIF](https://media1.giphy.com/media/jtUiNaYnqZZlu/giphy.gif)